### DISCUSSION TITLE
github pack

### DISCUSSION THREAD

**Discussion by faisaliqbal909090** [community/community#62384] [GitHub Education]
### Select Topic Area

Question

### Body

university has not issued me an email address,  so what  need to provide alternative documents to prove your current student status while applying for the GitHub Student Developer Pack.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @faisaliqbal909090 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
